test/testsuite.sh

```diff
@@ -28,15 +28,28 @@ echo "        TOMCAT_CYCLE_COUNT=$TOMCAT_CYCLE_COUNT"
 echo "        ITERATION_COUNT=$ITERATION_COUNT"
 echo "        IMG=$IMG"
 echo "        HTTPD_IMG=$HTTPD_IMG"
+if [ ! -z ${MPC_CONF+x} ]; then
+echo "        MPC_CONF=$MPC_CONF"
+fi
 
 if [ ! -d logs ]; then
     mkdir logs
 fi
 
 . includes/common.sh
 
-httpd_create  > /dev/null 2>&1 || exit 2
-tomcat_create > /dev/null 2>&1 || exit 3
+if [ -d target ]; then
```

Comment:
Shouldn't there be a `!`?

Reply:
Trying it on a fresh repository, it should be `! -d tomcat/target` actually.

Reply:
fixed